Helpers/opent2t-translator-helper-hue/js/index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var request = require('request-promise');
```

Comment:
Since you are adding the helper, I'd say don't add it since we are now using the hub.

---
In reply to: [81057804](https://github.com/openT2T/translators/pull/96#discussion_r81057804) [](ancestors = 81057804)